src/coprocessor/endpoint.rs

```diff
@@ -600,6 +600,11 @@ impl<E: Engine> Endpoint<E> {
             .or_else(|e| futures::future::ok(make_error_response(e))) // Stream<Resp, ()>
             .map(|item: std::result::Result<_, ()>| item.unwrap())
     }
+
+    /// Sets inner ResourceTagFactory of Endpoint.
+    pub fn set_resource_tag_factory(&mut self, rtf: ResourceTagFactory) {
+        self.resource_tag_factory = rtf;
+    }
```

Comment:
I guess this function is not needed

Reply:
I changed the test code to avoid relying on this setter method.